leku/src/main/java/com/schibstedspain/leku/geocoder/GeocoderPresenter.java

```diff
@@ -56,6 +59,7 @@ public void getFromLocationName(String query) {
         .subscribeOn(Schedulers.newThread())
         .observeOn(scheduler)
         .retry(RETRY_COUNT)
+        .onErrorResumeNext(apiInteractor.getFromLocationName(query))
```

Comment:
as it's a library is ok, i just think in a "future" when maybe another geocoder will be added.

It's ok, it can be refactored later :)